### DISCUSSION TITLE
How to get the elevation map?

### DISCUSSION THREAD

**Discussion by Seraen** [bulletphysics/bullet3#4097] [Q&A]
Hi, I am wondering how to get an elevation map from the environment, with the robot as the center and a size of 2*2 grids. Is there any function to implement?

**Comment by BluePrintRandom** [comment]
with a kdtree you can know what tile you are closest to, or the closest N tiles.

using A* you can get what tiles are connected and evaluate a cost function

**Reply by Seraen** [reply]
Thank you for answering! I would like to get the elevation information of the environment. Is there any built-in function that can do that?
